src/modules/30-secrets/components/CreateUpdateSecret/views/VaultFormFields.tsx

```diff
@@ -64,6 +87,19 @@ const VaultFormFields: React.FC<VaultFormFieldsProps & FormikContextProps<any>>
       {type === 'SecretFile' ? (
         <FormInput.FileInput name="file" label={getString('secrets.secret.labelSecretFile')} multiple />
       ) : null}
+      {secretManagerType === 'GcpSecretManager' &&
+        (formik?.values['valueType'] === 'Reference' ? (
+          <>
+            <FormInput.Text name="version" label={getString('version')} />
+          </>
```

Comment:
No need for a fragment here
```suggestion
          <FormInput.Text name="version" label={getString('version')} />
```